README.md

```diff
@@ -0,0 +1,20 @@
+# SDK Extension for Rust stable
+
+This extension contains various components of the [Rust](https://www.rust-lang.org) stable toolchain.
+
+
+## Mold
+
+In order to use the (fast) [mold](https://github.com/rui314/mold) linker:
+
+1. Add "org.freedesktop.Sdk.Extension.llvm12" next to this extension in order to get clang
```

Comment:
I would recommend using llvm13 here which is latest version on flathub

Reply:
Done